prog.cpp

```diff
@@ -2775,8 +2775,12 @@ bool compile_regression_tb(const std::string& name) {
 }
 
 std::vector<std::string> run_regression_tb(const std::string& name) {
-  //int res = system(string("g++ -fstack-protector-all -std=c++11 regression_tb_" + name + ".cpp " + name + ".cpp").c_str());
-  int res = cmd("g++ -fstack-protector-all -std=c++11 regression_tb_" + name + ".cpp " + name + ".cpp");
+#ifndef CGRAFLOW
```

Comment:
I add this to make default clockwork pass. Could you check if this work on your end. I also change the user_setting.sh

Reply:
This change works for me.